op-program/host/config/config.go

```diff
@@ -97,7 +97,7 @@ type Config struct {
 }
 
 func (c *Config) Check() error {
-	if !c.InteropEnabled && c.L2ChainID == 0 {
+	if !c.InteropEnabled && c.L2ChainID == eth.ChainIDFromUInt64(0) {
```

Comment:
nit: Could this also work?
```suggestion
	if !c.InteropEnabled && c.L2ChainID == (eth.ChainID{}){
```